include/seastar/core/internal/pollable_fd.hh

```diff
@@ -47,6 +48,10 @@ class packet;
 
 }
 
+struct pollable_fd_abort: public std::runtime_error {
```

Comment:
It's isn't exactly an (runtime) error.
Please rename to `pollable_fd_aborted`, add space before `:`, and derive it from `abort_requested_exception` so they could be handled in the same error category.

With that, you can override `what()` (inline here is fine):
```
    virtual const char* what() const noexcept override {
        return "pollable fd aborted";
    }
```

Reply:
Yes, I agree. This is much better! I fixed this.